platforms/jvm/scala/build.gradle.kts

```diff
@@ -34,9 +34,7 @@ dependencies {
     implementation(project(":process-services"))
     implementation(project(":toolchains-jvm"))
 
-    implementation(libs.groovy)
     implementation(libs.guava)
-    implementation(libs.inject)
```

Comment:
Dependencies `libs.groovy` and `libs.inject` are included transitively via `project(":core-api")`:
* https://github.com/gradle/gradle/blob/release/platforms/jvm/scala/build.gradle.kts#L21
* https://github.com/gradle/gradle/blob/release/subprojects/core-api/build.gradle.kts#L35
* https://github.com/gradle/gradle/blob/release/subprojects/core-api/build.gradle.kts#L39